simulationworkflowschema/molecular_dynamics.py

```diff
@@ -1039,6 +1294,10 @@ class MolecularDynamicsResults(ThermodynamicsResults):
         sub_section=MeanSquaredDisplacement, repeats=True
     )
 
+    free_energy_calculations = SubSection(
```

Comment:
Just a silly question, related with the English and which is also bothering me when defining the base classes: if this is a singular MSection, shouldn't we be using singular? Then, the SubSection, if repeated, is in plural. Like (note the singular and plural):
```
free_energy_calculations = SubSection(FreeEnergyCalculation, repeats=True)
```

Or, if it does not repeat:
```
free_energy_calculation = SubSection(FreeEnergyCalculation)
```